tests/scripts/test_scripts.py

```diff
@@ -232,93 +225,84 @@ def test_train_dagger_main(tmpdir):
     assert isinstance(run.result, dict)
 
 
-def test_train_dagger_warmstart(tmpdir):
+def test_train_bc_main(tmpdir):
```

Comment:
This seems like it's testing a lot less than before -- why? Are we no longer able to warmstart runs in DAgger?

Reply:
I added back the test here: [8c3a7a9](https://github.com/HumanCompatibleAI/imitation/pull/445/commits/8c3a7a95c5c700e5a52badee148ec61e448bf167)